rt_bene_model_training/dataset_manager.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import cv2
+import numpy as np
+from os import listdir
+
+import json
+import csv
+from tqdm import tqdm
+import itertools
+
+
+def read_rgb_image(img_path, size, flip):
+    assert type(size) is tuple, "size parameter must be a tuple, (96, 96) for instance"
+    img = cv2.imread(img_path, cv2.IMREAD_COLOR)
+    if img is None:
+        print("ERROR: can't read " + img_path)
+    if flip:
+        img = cv2.flip(img, 1)
+    img = cv2.resize(img, size, cv2.INTER_CUBIC)
+    return img
+
+
+def load_one_flipped_pair(l_path, r_path, size):
+    l_img = read_rgb_image(l_path, size, False)
+    r_img = read_rgb_image(r_path, size, True)
+    return l_img, r_img
+
+class RTBeneDataset(object):
+    def __init__(self, csv_subjects, input_size):
+        self.csv_subjects = csv_subjects
+        self.input_size = input_size
+        self.subjects = {}
+        self.folds = {} 
+
+    def load_one_subject(self, csv_labels, left_folder, right_folder):
+        subject = {}
+        subject['y'] = []
+
+        left_inputs = []
+        right_inputs = []
+
+        with open(csv_labels) as csvfile:
+            csv_rows = csv.reader(csvfile)
+            for row in tqdm(csv_rows):
+                img_name = row[0]
+                img_lbl = float(row[1])
+                if img_lbl == 0.5:
```

Comment:
Add brief comment that `img_lbl == 0.5` means that it is uncertain whether the eyes are open or not, so we skip the image.